keras_retinanet/backend/common.py

```diff
@@ -20,7 +20,12 @@
 import numpy as np
 
 
-def bbox_transform_inv(boxes, deltas):
+def bbox_transform_inv(boxes, deltas, mean=None, std=None):
+    if mean is None:
+        mean = [0, 0, 0, 0]
+    if std is None:
+        std = [0.1, 0.1, 0.2, 0.2]
```

Comment:
I think this approach is good for now. Computing the statistics on the dataset adds some unnecessary complexity.